lib/web/app/redirect.go

```diff
@@ -73,8 +77,8 @@ const js = `
           body: JSON.stringify(data),
         }).then(response => {
           if (response.ok) {
-            // redirect to the root and remove current URL from history (back button)
-            window.location.replace('/');
+            // redirect to the target path and remove current URL from history (back button)
+            window.location.replace(path || "/");
```

Comment:
Yeah, great call out. Oversight on my part. Replaced with a built url from origin + path, with error catching to return to path only just in case. No redirects to outside sources now